src/menutemplate.js

```diff
@@ -135,6 +135,20 @@ var MenuTemplate = function () {
           label: 'Toggle Chromium Developer Tools',
           accelerator: 'Alt+' + util.CommandOrCtrl() + '+I',
           click: function() { remote.getCurrentWindow().toggleDevTools(); }
+        },
+        {
+          label: 'Log Zoom In',
+          accelerator: util.CommandOrCtrl() + '+=',
```

Comment:
Can this say `++` ?
In my app I get this:
![zoom](https://cloud.githubusercontent.com/assets/3965324/14752506/78a169da-0884-11e6-9b42-2110bea524d3.png)